web/src/components/netflow-record/record-panel.tsx

```diff
@@ -0,0 +1,142 @@
+  id?: string;
+};
+
+export const RecordPanel: React.FC<RecordDrawerProps> = ({
+  id,
+  record,
+  columns,
+  filters,
+  range,
+  options,
+  setFilters,
+  setRange,
+  setQueryOptions,
+  onClose
+}) => {
+  const { t } = useTranslation('plugin__network-observability-plugin');
+
+  const onClick = React.useCallback(
+    (col: Column, isDelete: boolean) => {
+      if (!record) {
+        return;
+      }
+
+      if (isDelete) {
+        switch (col.id) {
+          case ColumnsId.timestamp:
+            setRange(DEFAULT_TIME_RANGE);
+            break;
+          case ColumnsId.flowdir:
+            setQueryOptions({ ...options, reporter: flowdirToReporter[DEFAULT_FLOWDIR] });
+            break;
+          default:
+            setFilters(filters.filter((f: Filter) => f.colId !== col.id));
+            break;
+        }
+      } else {
+        const value = col.value(record);
+        switch (col.id) {
+          case ColumnsId.timestamp:
+            //Filter at exact same date in ms
+            const dateSeconds = getDateMsInSeconds(Number(value));
+            setRange({ from: dateSeconds, to: dateSeconds + 1 });
+            break;
+          case ColumnsId.flowdir:
+            setQueryOptions({ ...options, reporter: flowdirToReporter[value] });
+            break;
+          default:
+            const values = [{ v: value.toString() }];
+            const result = _.cloneDeep(filters);
+            const found = result.find(f => f.colId === col.id);
+            if (found) {
+              found.values = values;
+            } else {
+              result.push({ colId: col.id, values: values });
+            }
+            setFilters(result);
+            break;
+        }
+      }
+    },
+    [filters, options, record, setFilters, setQueryOptions, setRange]
+  );
+
+  const getFilter = (col: Column) => {
+    let isDelete = false;
+    if (record) {
+      const value = col.value(record);
+      switch (col.id) {
+        case ColumnsId.timestamp:
+          isDelete = typeof range !== 'number' && range.from === getDateMsInSeconds(Number(value));
+          break;
+        case ColumnsId.flowdir:
+          isDelete = options.reporter === flowdirToReporter[value];
+          break;
+        default:
+          isDelete =
+            filters.find((f: Filter) => f.colId === col.id && f.values.find(v => v.v === value.toString())) !==
+            undefined;
+          break;
+      }
+    }
+    return {
+      onClick,
+      isDelete
+    };
+  };
+
+  return (
+    <DrawerPanelContent id={id}>
+      <DrawerHead>
+        <Text component={TextVariants.h2}>{t('Traffic details')}</Text>
+        <DrawerActions>
+          <DrawerCloseButton onClick={onClose} />
+        </DrawerActions>
+      </DrawerHead>
+      <DrawerPanelBody>
+        {record &&
+          columns.map(c => (
+            <TextContent className="record-field-container" key={c.id}>
+              <Text component={TextVariants.h3}>{c.name}</Text>
```

Comment:
I would say `h3` is too big here (especially if we add more fields as mentioned above), it makes it necessary to scroll to see everything

Reply:
Does `h4` fits better for you ?
![image](https://user-images.githubusercontent.com/91894519/152178904-d3f916e2-2f64-40f9-90a9-ce7dce9716b0.png)